includes/polling/applications/docker.inc.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use LibreNMS\Exceptions\JsonAppException;
+use LibreNMS\Exceptions\JsonAppMissingKeysException;
+use LibreNMS\RRD\RrdDefinition;
+
+$name = 'docker';
+$app_id = $app['app_id'];
+$output = 'OK';
+
+function convertToBytes(string $from): ?int
```

Comment:
Use Number::formatBi() isntead ;)

Reply:
Ok how to use this ?

Reply:
The source code in LibreNMS/Util/Number.php shows how